bot/util/playerUtils.js

```diff
@@ -29,46 +29,46 @@ async function generatePlayerCard (player, forbidden) {
 
         playerCard
             .addField("Tournament Team", `${
-                emojiString(tourneyTeam.emoji)} ${tourneyTeam.fullName} (${
-                getPosition(tourneyTeam, player)})`, true);
+                emojiString(tourneyTeam.emoji)} ${tourneyTeam.fullName}\n- ${
+                getPosition(tourneyTeam, player)}`, true);
     
     }
 
-    if (forbidden) {
-
-        playerCard.addField("Fingers", `||${player.totalFingers} Fingers||`, true);
-
-    }
     playerCard
+        .addField("Current Vibe", player.permAttr.includes("SCATTERED")
+            ? (forbidden ? `||${vibeString(vibes(player))}||` : "** **")
+            : vibeString(vibes(player)), true)
+        .addField("Evolution", (player.evolution > 0 && player.evolution < 4)
+            ? `**Base ${player.evolution}**`
+            : (player.evolution === 4 ? "Home" : "Base"), true)
         .addField(
             "Peanut Allergy",
-            player.peanutAllergy ? "🤮" : "😋", true
+            player.peanutAllergy ? "🤢" : "😋", true
         )
-        .addField("Fate", player.fate ?? "A roll of the dice", true)
-        .addField("Evolution", ((player.evolution > 0 && player.evolution < 4)
-            ? `**Base ${player.evolution}**`
-            : (player.evolution === 4 ? "Home" : "Base")), true);
+        .addField("Pregame Ritual", player.ritual || "** **", true)
+        .addField("Coffee Style", player.coffee !== null
+            ? (player.coffee === 7 ? "Espresso" : await coffeeCache.fetch(player.coffee))
```

Comment:
Leave espresso as "coffee?", it is an easteregg to the fact espresso didn't work for a long time